Makefile

```diff
@@ -42,10 +42,10 @@ test-client:
 	yarn --cwd client test
 
 test-server:
-	pipenv run pytest -n 2
+	pipenv run pytest
 
 test-server-coverage:
-	pipenv run pytest -n 2 --cov=.
+	pipenv run pytest --cov=.
```

Comment:
@umbernhard heads up - had to remove the parallelism because it's causing intermittent failures again